javascript/asynchronous_javascript_and_apis/project_weather_app.md

```diff
@@ -1,34 +1,34 @@
 ### Introduction
 
-Use everything we've been discussing to create a weather forecast site using the weather API from the previous lesson. You should be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.
+Use everything we've been discussing to create a weather forecast site using the Visual Crossing API from the previous lesson. You should be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.
 
 You should change the look of the page based on the data, maybe by changing the color of the background or by adding images that describe the weather. (You could even use the Giphy API to find appropriate weather-related gifs and display them). Feel free to use promises or async/await in your code, though you should try to become comfortable with both.
 
 <div class="lesson-note" markdown="1">
-  
-#### WeatherAPI free tier
 
-After creating your API key from WeatherAPI, you will automatically be moved to the free tier after 14 days. This will limit the forecast data available to you to 3 days. You may want to keep this in mind while making design choices for the forecast display.
+#### Visual Crossing free tier
+
+The free tier of the Visual Crossing API provides 1000 queries. This is what "queryCost" means in the returned JSON.
 
 </div>
```

Comment:
On second thought, I don't think we really need to mention this. I don't think many people will actually encounter the 1000 daily records limit, and if someone does, it's unlikely to be a consistent problem across many learners. The original note box was specifically about a quirk of WeatherAPI's free tier process affecting features. Visual Crossing doesn't have this quirk, so should be fine to leave as is.